processor/configuration/src/main/resources/schemas/actions/webhook_sink_0.1.json

```diff
@@ -35,5 +39,13 @@
     "basic_auth_username",
     "basic_auth_password",
     "ssl_verification_disabled"
-  ]
-}
\ No newline at end of file
+  ],
+  "dependentRequired": {
+    "basic_auth_username": [
+      "basic_auth_password"
+    ],
+    "basic_auth_password": [
+      "basic_auth_username"
+    ]
```

Comment:
I have a question to check my understanding. This configuration snippet forces user to provide username and password if given authentication method was selected.

Shouldn't we do sth similar for S3:
```
    "aws_uri_endpoint_override": {
      "title": "Overwrite Endpoint URI",
      "description": "Set the overriding endpoint URI. This option needs to be used in combination with overrideEndpoint option.",
      "type": "string"
    },
    "aws_override_endpoint": {
      "title": "Endpoint Overwrite",
      "description": "Set the need for overiding the endpoint URI. This option needs to be used in combination with uriEndpointOverride setting.",
      "type": "boolean",
      "default": false
    },
```

Reply:
makes sense

Reply:
https://issues.redhat.com/browse/MGDOBR-886

Reply:
will do in https://issues.redhat.com/browse/MGDOBR-886